spring-boot/admin/src/test/resources/application-test.yml

```diff
@@ -0,0 +1,44 @@
+spring:
+  data:
+    rest:
+      base-path: /api
+logging:
+  file: /tmp/coatrack.log
+  level:
+    nl: DEBUG
+    org: INFO
+    com: DEBUG
+ygg:
+  admin:
+    server:
+      url: http://localhost:8080
+    servicecovers:
+      path: /tmp
+      url: http://localhost/coatrack-servicecovers/
```

Comment:
General comment: there are several occurrences of `localhost` in this file, maybe replace them by something like `example-url.coatrack.eu`

Reply:
Done and Working well